src/pages/components/GeneralOptions.tsx

```diff
@@ -28,12 +30,48 @@ const fp: [string, FilenamePatternToken][] = [
 ]
 
 const TokenPanel = memo(
-  ({ handleTokenToggle, pattern, previewFilename }: TokenPanelProps) => {
+  ({ handleTokenToggle, handleTokenSort, pattern, previewFilename }: TokenPanelProps) => {
     return (
       <>
         <Flex minH={'1.5em'} fontSize="1.2em">
           {previewFilename}
         </Flex>
+        <DragDropContext
+          onDragEnd={({ source, destination }) =>
+            handleTokenSort(source.index, destination.index)
```

Comment:
`destination` will be `null`, if the order wasn't change.
This condition should be handled.
```js
destination && handleTokenSort(source.index, destination.index)
```